script/upgrades/MUGOV/MUGOVChecks.sol

```diff
@@ -86,9 +98,12 @@ contract MUGOVChecks is GovernanceScript, Test {
     assertEq(airgrab.token(), address(mentoToken), "Airgrab token is incorrect");
     assertEq(airgrab.locking(), address(locking), "Airgrab locking is incorrect");
     assertEq(airgrab.mentoTreasury(), address(governanceTimelock), "Airgrab Mento Treasury is incorrect");
+
+    // @TODO: Replace below check with 2 weeks after updating mento-core
+    assertEq(airgrab.endTimestamp() - block.timestamp, 365 days, "Airgrab end timestamp is incorrect");
```

Comment:
I am not sure if the strict equality will hold here, assuming the checks will run one block after the deployment

Reply:
both the deployment + checks all happen on the same block